cmd/lncli/types.go

```diff
@@ -11,7 +11,7 @@ type OutPoint string
 
 // NewOutPointFromProto formats the lnrpc.OutPoint into an OutPoint for display.
 func NewOutPointFromProto(op *lnrpc.OutPoint) OutPoint {
-	return OutPoint(fmt.Sprintf("%s:%d", op.TxidStr, op.OutputIndex))
+	return OutPoint(fmt.Sprintf("%x:%d", op.TxidBytes, op.OutputIndex))
```

Comment:
Will this change break any of the other cli commands?

Reply:
Don't think so, I've made sure that the `TxidBytes` is always set for `lnrpc.OutPoint`s.